src/components/toast/toast.pw.tsx

```diff
@@ -320,4 +321,21 @@ test.describe("Accessibility tests for Toast component", () => {
 
     await checkAccessibility(page);
   });
+
+  test("should check accessibility more than one Toast is rendered with undefined content on close and defined on open", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ToastWithConditionalContent />);
+
+    // check accessibility when both toasts are closed and have undefined content
+    await checkAccessibility(page);
+
+    await button(page).nth(0).click();
+
+    await page.waitForTimeout(1000);
```

Comment:
**suggestion(non-blocking):** Rather than having a `waitforTimeout` here, you should be able to add `toastComponent(page)` or another locator for Toast to the `checkAccessibility` function. Something like `await checkAccessibility(page, toastComponent(page));` should only run the accessibility checks once the locator is found. 

We also have the option of using the `waitForAnimationEnd()` function too, which is what the `checkAccessibility` function _should_ do.
` await waitForAnimationEnd(Toast locator goes here);`.

If that doesn't work, [the Toast animation transition is 300ms](https://github.com/Sage/carbon/blob/master/src/components/toast/toast.style.ts#L153) so you might be able to reduce the `1000` value to `300`.